config/helm/chart/default/templates/Common/csi/priority-class.yaml

```diff
@@ -0,0 +1,24 @@
+{{- include "dynatrace-operator.platformRequired" . }}
+{{ if eq (include "dynatrace-operator.needCSI" .) "true" }}
+
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+kind: PriorityClass
+apiVersion: scheduling.k8s.io/v1
+metadata:
+  name: dynatrace-high-priority
+value: {{ int .Values.csidriver.priorityClassValue }}
+globalDefault: false
+description: "This priority class is used for Dynatrace Components with a higher priority"
```

Comment:
we could provide more reason why we need this, e.g. "... to make sure the pods are terminated in the correct order"

Reply:
yes and not, what if customers want to reuse it on oneAgent pods? Or we reuse it in the future for operator/webhook pods aswell?